_episodes/12-cmdline.md

```diff
@@ -79,8 +82,11 @@ We'll tackle these questions in turn below.
 
 ## Command-Line Arguments
 
-Using the text editor of your choice,
-save the following in a text file called `sys_version.py`:
+Using the text editor of your choice, we can
+write a simple script that will tell us what version of
+python we are using. The sys library contains variables and functions
+associated with the interpreter.
+Save the following in a text file called `sys_version.py`:
```

Comment:
I think it makes sense to give a quick explanation of what the sys_version.py script will do as you have added here. I suggest a few edits: removing the word "simple" in case it is demotivating for anyone who does not consider this is simple,  capitalize Python, move the description of `sys` library to where it is talked about a little later and tweak it slightly. What do you think about something like:
```
Using the text editor of your choice, let's write a
short script that tells us what version of Python we are using.
Save the following in a text file called `sys_version.py`:
```
Then after the code:
```
The first line imports a library called `sys`, which is short for “system”.
`sys` provides access to variables and functions associated with the interpreter.
It defines values...
```
```

Reply:
I agree.